Devscord.DiscordFramework/Workflow.cs

```diff
@@ -125,5 +128,14 @@ private async Task CallUserJoined(SocketGuildUser guildUser)
 
             OnUserJoined.ForEach(x => x.Invoke(contexts));
         }
+
+        private async Task CallServerAddedBot(SocketGuild guild)
+        {
+            var discordServerFactory = new DiscordServerContextFactory();
```

Comment:
może jest opcja żeby to przychodziło z IoC
jeśli nie to może zostać
ale wolałbym żeby klasy które można brać z IoC, były brane z IoC
wtedy nie będzie problemu kiedy taka klasa będzie musiała brać jakieś inne zależności 
(nie okaże sie że nagle mamy problem)

Reply:
Nie za bardzo, trzeba by przekazywać ręcznie każdą klasę jako argument, więc chyba już lepiej tak zostawić